build.rs

```diff
@@ -1,11 +1,14 @@
 #[cfg(target_arch = "x86_64")]
-extern crate cc;
-
 fn main() {
-    if cfg!(target_arch = "x86_64") {
+    let target_arch = std::env::var("CARGO_CFG_TARGET_ARCH").unwrap();
+
+    if target_arch == "x86_64" {
```

Comment:
This addition is not needed as we ensured that the target arch is `x86_64` by the cfg flag.

Reply:
it is very much needed, as it covers the case of cross compiling, the env variable is the actual target, and the cfg is the host target

Reply:
Nice cfg-fu.